src/common/AndroidTypes.ts

```diff
@@ -147,148 +146,3 @@ export class AndroidPackage {
         }`;
     }
 }
-
-export class AndroidVirtualDevice {
```

Comment:
These are not needed anymore since they've been refactored into `AndroidDevice` and `AndroidDeviceManager`